src/fsharp/MethodCalls.fs

```diff
@@ -254,25 +254,25 @@ type CalledMeth<'T>
                     | None -> true)
 
             // See if any of them are 'out' arguments being returned as part of a return tuple 
-            let unnamedCalledArgs, unnamedCalledOptArgs, unnamedCalledOutArgs = 
+            let minArgs, unnamedCalledArgs, unnamedCalledOptArgs, unnamedCalledOutArgs = 
                 let nUnnamedCallerArgs = unnamedCallerArgs.Length
-                if allowOutAndOptArgs && nUnnamedCallerArgs < unnamedCalledArgs.Length then
+                let nUnnamedCalledArgs = unnamedCalledArgs.Length
+                if allowOutAndOptArgs && nUnnamedCallerArgs < nUnnamedCalledArgs then
```

Comment:
maybe replace with

```fsharp
nUnnamedCallerArgs - 1,
if // ... existing code
```

this would minimize the diff and make sure we always return that same expression which probably should be the same at all times.

Reply:
Line 258 and 259 are actually identical:

````
            let minArgs, unnamedCalledArgs, unnamedCalledOptArgs, unnamedCalledOutArgs = 

                let nUnnamedCallerArgs = unnamedCallerArgs.Length

                let nUnnamedCalledArgs = unnamedCalledArgs.Length

                if allowOutAndOptArgs && nUnnamedCallerArgs < nUnnamedCalledArgs then

````

It looks like you have this line twice, @ 258 and 259:

https://github.com/forki/visualfsharp/blob/912e00b0cf1c7506d0d0fa167dd75db8dac72e72/src/fsharp/MethodCalls.fs#L258